app/code/Magento/AdminNotification/Test/Integration/Model/ResourceModel/Inbox/Collection/CriticalTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\AdminNotification\Test\Integration\Model\ResourceModel\Inbox\Collection;
+
+use Magento\AdminNotification\Model\ResourceModel\Inbox\Collection\Critical;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class CriticalTest extends TestCase
+{
+    /**
+     * @var Critical
+     */
+    protected $model;
```

Comment:
Please make it private 
```suggestion
    private $model;
```